src/Widgets/AppEntry.vala

```diff
@@ -37,13 +38,19 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
         var notification = entry.notification;
         app_info = notification.app_info;
 
-        var label = new Gtk.Label (app_info.get_name ());
+        string name = _("Other");
```

Comment:
If `app_info` is unlikely to be NULL, then it's maybe better to assign other in an `else` clause (otherwise we are allocating a string to free it right after)